### PR TITLE
Simplify integration test

### PR DESCRIPTION
**Pull request by mitchelldyer01**
This PR simplifies the already simple integration test

### CODE REVIEW

File: integration/integration_test.go [hunk 1]
```diff
@@ -74,9 +62,17 @@ func Test_Main(t *testing.T) {
 		Status(200).
 		Done()
 
-	// GET, PUT, POST should pass with an auth token
 	for _, tc := range tcs {
 		t.Run(fmt.Sprintf("%s%s", tc.method, tc.path), func(t *testing.T) {
+			// GET, PUT, POST should fail if there's no auth token
```
Comment (dylanhitt) [inline integration/integration_test.go:67]: Thoughts in separating the tables out by pass and by failure so 

```
for _, tc := range tcs {
		t.Run(fmt.Sprintf("%s%s_failures", tc.method, tc.path), func(t *testing.T) {
                 .....
                 t.Run(fmt.Sprintf("%s%s", tc.method, tc.path), func(t *testing.T) {
```

**Review by dylanhitt — COMMENTED** [review]
Looks fine. Just that one thing

**Comment by dylanhitt** [comment]
Do you plan on adding this into CI somehow?

**Comment by mitchelldyer01** [comment]
> Do you plan on adding this into CI somehow?

Yeah. It actually gets bootstrapped in `docker` if you run `make test`

File: integration/integration_test.go (hunk 1 above)
Comment (mitchelldyer01) [reply integration/integration_test.go:68]: Good idea. Originally I had them split out by loop, but splitting them about test case is the best of both worlds.

**Pull request merged**